### PR TITLE
fixes for stCallCodes/callcall_00

### PR DESCRIPTION
**Pull request by cdetrio**
Part of #184.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -51,10 +51,12 @@ defaults:
       # git fetch origin evm2wasm-trace
       # git checkout evm2wasm-trace
       command: |
-        git clone https://github.com/ethereum/cpp-ethereum --branch ewasm-json-trace --single-branch
+        git clone https://github.com/ethereum/cpp-ethereum --branch develop --single-branch
```
Comment (axic) [inline circle.yml]: Should be using the `ewasm` branch IIRC.

Comment (cdetrio) [reply circle.yml]: `ewasm` branch last commit was 23 days ago. I've been using `develop` locally and it works.

Comment (axic) [reply circle.yml]: Just rebase `ewasm` against `develop` and push it. The last decision was it is kept up to date. However I do not think there were any relevant changes on `develop`.

**Comment by cdetrio** [comment]
The cpp-ethereum build step kept failing with this error:
```
[ 72%] Building CXX object test/CMakeFiles/testeth.dir/tools/jsontests/BlockChainTests.cpp.o
[ 73%] Building CXX object test/CMakeFiles/testeth.dir/tools/jsontests/StateTests.cpp.o
[ 73%] Building CXX object test/CMakeFiles/testeth.dir/tools/jsontests/TransactionTests.cpp.o
c++: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-6/README.Bugs> for instructions.
eth/CMakeFiles/eth.dir/build.make:86: recipe for target 'eth/CMakeFiles/eth.dir/main.cpp.o' failed
make[2]: *** [eth/CMakeFiles/eth.dir/main.cpp.o] Error 4
CMakeFiles/Makefile2:1043: recipe for target 'eth/CMakeFiles/eth.dir/all' failed
make[1]: *** [eth/CMakeFiles/eth.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

It started building successfully again at commit d84ae51 "try develop 2". I also updated the `node-cpp-build-env` docker image before that though, so not exactly sure what fixed it.

File: wasm/generateInterface.js [hunk 2]
```diff
@@ -370,16 +370,27 @@ function generateManifest (interfaceManifest, opts) {
         call += '(get_local $callback)'
       }
 
+      if (opcode === 'CALL') {
+        call =
+          `(i64.store
+      (i32.add (get_global $sp) (i32.const ${spOffset * 32}))
+      (i64.extend_u/i32
+        (i32.xor (i32.const 1) ${call}) ;; flip CALL result from EVM convention to POSIX convention, 0 -> 1, 1 -> 0
```
Comment (axic) [inline wasm/generateInterface.js]: This is not enough, because it could return 2, which is xor'd into 3.

**Comment by lrettig** [comment]
Note that this partially overlaps with #196 e.g. https://github.com/ewasm/evm2wasm/pull/196/files#diff-e7aee89bb9a80fcf0c9c6ade2d73b4c8R125

**Pull request closed without merging**